Shared/IdentifyController.cpp

```diff
@@ -265,36 +118,50 @@ bool IdentifyController::addGeoElementPopup(GeoElement* geoElement, const QStrin
     return false;
 
   // create a new Popup from the geoElement
-  Popup* newPopup = new Popup(geoElement, this);
+  std::unique_ptr<Popup> newPopup{new Popup(geoElement, this)};
```

Comment:
Not recommended to mix ownership between QObject parents and unique_ptr. Typically one wins (unique_ptr goes first), and then QObject is resilient against double deletes, but we should be explicit with ownership intent.


```suggestion
  std::unique_ptr<Popup> newPopup = std::make_unique<Popup>(geoElement);
```

Reply:
Understood. Will switch to std::make_unique.